package.json

```diff
@@ -35,7 +32,6 @@
     "tool-bar": {
       "description": "A container for buttons at workspace edge",
       "versions": {
-        "0.1.0": "provideToolBarLegacy",
         "1.0.0": "provideToolBar"
```

Comment:
Maybe I'm misunderstanding what you meant by "merge". Are you saying:

```
"tool-bar": {
  "versions": {
    "0.1.0": "provideToolBar",    
    "1.0.0": "provideToolBar"
  }
}
```

That's kinda dangerous, since you don't know what kinds of private packages people have